configs/common/modules.yaml

```diff
@@ -212,6 +226,10 @@ modules:
         suffixes:
           ^esmf@8.2.0~debug: 'esmf-8.2.0'
           ^esmf@8.2.0+debug: 'esmf-8.2.0-debug'
+      openmpi:
+        environment:
+          set:
+            'OMPI_MCA_rmaps_base_oversubscribe': '1'
```

Comment:
Nice, this was always one of the reasons I used MPICH. Having it encoded into the environment is good.

Reply:
Problem with mpich on macOS is that once the ufs-weather-model update to use mpi_f08 is made, we can't use mpich on macOS anymore, because it doesn't build mpi_f08 on macOS. But openmpi does. So I am glad that there is a solution to (a) the oversubscribe problem and (b) the two-level namespace problem.